lib/facter/facts/azurelinux/os/release.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Facts
+  module Azurelinux
+    module Os
+      class Release
+        FACT_NAME = 'os.release'
+        ALIASES = %w[operatingsystemmajrelease operatingsystemrelease].freeze
+
+        def call_the_resolver
+          version = from_specific_file || from_os_release
+
+          return Facter::ResolvedFact.new(FACT_NAME, nil) unless version
+
+          [Facter::ResolvedFact.new(FACT_NAME, version),
+           Facter::ResolvedFact.new(ALIASES.first, version['major'], :legacy),
+           Facter::ResolvedFact.new(ALIASES.last, version['full'], :legacy)]
```

Comment:
Can you add support for minor?

Reply:
I think this is still an open question?

Reply:
We're actually getting minor in the facter output already, but I'm not sure how that's working.